src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -18,6 +18,10 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available on this platform");
         }
 
+        if ( SystemUtils.OS_ARCH.equals("ppc64le") ) {
```

Comment:
Could you extract an `isPPCArchitecture()` method, add it to a new `NativeUtils` class in the `org.broadinstitute.hellbender.utils` package, and call that method from here? I think we might want to do this check in more than one place in the GATK.